crypto/encode_decode/decoder_pkey.c

```diff
@@ -416,10 +467,8 @@ int ossl_decoder_ctx_setup_for_pkey(OSSL_DECODER_CTX *ctx,
     }
 
     ok = 1;
- err:
+err:
```

Comment:
Looks to be an accidental change, likely due to the fact that diff has been producing unhelpful output.

There doesn't appear to be a clear standard here; I count 886 labels in libcrypto with a space, and 151 without. Since labels with a space are more numerous, is this intended to be the convention?

Reply:
Heh, I suspect different editors will give different results for labels...

Our cc-mode style for emacs stipulates that extra space:

https://github.com/openssl/openssl/blob/1483b37e7a2c952eed5f6c7f5c0be9635aa3a6ea/doc/openssl-c-indent.el#L50

... as does the `indent` profile that we used for source reformatting all those years ago:

https://github.com/openssl/openssl/blob/1483b37e7a2c952eed5f6c7f5c0be9635aa3a6ea/util/indent.pro#L15